webapp/src/monacoSnippets.ts

```diff
@@ -566,6 +566,7 @@ function cachedBuiltinCategories(): pxt.Map<BuiltinCategoryDefinition> {
                 theEditor.showPackageDialog();
                 return true;
             },
+            onlyTriggerOnClick: true,
```

Comment:
I had to fix this in the same PR as other changes resulted in not being able to close the extensions page/window when using JavaScript.